Unwrap/Activities/Home/HomeDataSource.swift

```diff
@@ -215,6 +210,17 @@ class HomeDataSource: NSObject, UITableViewDataSource {
         }
     }
 
+    /// Dequeue a reusable and clean table view cell to show an stat.
+    func dequeueStatReusableCell(in tableView: UITableView, indexPath: IndexPath) -> UITableViewCell {
```

Comment:
I'm not so sure about if this should be done this way or even if this name is the best.
Suggestions are appreciated.

Reply:
I don't see a problem with this approach – I like it!